src/code/utilities/lang/en-us.js

```diff
@@ -62,5 +62,12 @@ export default {
   "~COUNTER.n_OF_N": "${0}\xA0\xA0of\xA0\xA0${1}",
 
   // Messages from ITS
-  "~ITS.GREETING": "Hi there user!"
+  "~ITS.GREETING": "Hi there user!",
+
+  // VenturePad
+  "~VENTURE.END_LEVEL": "CHALLENGE COMPLETE",
+  "~VENTURE.YOU_EARNED": "You've earned",
+  "~VENTURE.AMBER_GEM": "Amber Crystal",
+  "~VENTURE.RED_GEM": "Red Crystal",
+  "~VENTURE.BLUE_GEM": "Blue Crystal"
```

Comment:
Another minor comment: I like how the css classes just use `.gem-fill-0`, `.gem-fill-1` etc. It seems like it would be nice to do the same thing with the award string keys: `~VENTURE.AWARD_FIRST`, `~VENTURE.AWARD_SECOND` or something. That way, if/when the GV team decides that they want to reorder to colors, or use different colors, you just need to change the CSS file and the Strings file, instead of also changing the keys everywhere in the code.

Again, not really important.